server/server.js

```diff
@@ -1,12 +1,39 @@
 const express = require('express');
 const db = require('../database/database.js');
+const seed = require('../database/seed.js');
+
+const faker = require('faker');
 
 const app = express();
 const PORT = 3003;
 
 app.use('/games/:gameId', express.static(`${__dirname}/../public`));
 app.use(express.json());
 
+
+// Create
+
+app.post('/post', (req, res) => {
+  let newRecord = {
+    gameTitle: faker.name.title(),
+    newPrice: 59.99,
+    preOwnedPrice: 54.99,
+    digitalPrice: 59.99,
+    mainImage: seed.randomMainImage(),
+    images: seed.randomImages(),
+  }
```

Comment:
As mentioned in your put request, I'd suggest generating the data outside of this api request handler so you also have the freedom to send in data of your choosing.

Reply:
I hear you. I did this for the same reason I did the PUT one.